tests/gameBoardModule.test.js

```diff
@@ -0,0 +1,24 @@
+import { gameBoardModule } from '../main';
+
+describe('checking gameboard', () => {
+  const board = ['', '', '', '', '', '', '', '', ''];
+  const newBoard = ['', 'X', '', '', '', '', '', '', ''];
+
+  test('should check for winning combos', () => {
+    expect(gameBoardModule.checkWinCombo()).toBe(null);
+  });
+
+  test('check if the there is a gameboard', () => {
+    expect(gameBoardModule.boardArr).toEqual(board);
+  });
+
+  test('check if the reset board is correct', () => {
+    gameBoardModule.boardArr[1] = 'X';
+    expect(gameBoardModule.boardArr).toEqual(newBoard);
+  });
+
+  test('check if the reset board is correct', () => {
+    gameBoardModule.boardArr[1] = 'X';
+    expect(gameBoardModule.boardArr).toContain('X');
+  });
```

Comment:
- [ ] I'm confused about how these 2 tests are checking for a reset board. Isn't the reset board functionality already been checked in the other test module? I think these 2 tests are not doing much, we can think in other things that we can test in your application, like if the player's names are been properly displayed in their turn. The ideal scenario is to test all the features of your application

Reply:
My bad! Neither of these is meant to reset the board. These two were meant to check if the board is being rendered correctly after an item is placed on the board.